internal/api/identity.go

```diff
@@ -0,0 +1,8 @@
+package api
+
+import "net/http"
+
+func (a *API) IdentityDelete(w http.ResponseWriter, r *http.Request) error {
```

Comment:
Might be missing an implicit use case but is this still WIP/in use? Couldn't find where it's used

Reply:
oops you're right, i accidentally added it, will remove